svg/elements/stop.json

```diff
@@ -172,9 +173,15 @@
               "safari": {
                 "version_added": "3"
               },
-              "safari_ios": {
-                "version_added": "3"
-              },
+              "safari_ios": [
+                {
+                  "version_added": "3",
+                  "version_removed": "11"
```

Comment:
@ddbeck did you mean to suggest putting this entry first? Now https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stop-opacity#browser_compatibility makes it look like this isn't supported on iOS at all.

Reply:
No, I did not. I've opened a PR to correct this: https://github.com/mdn/browser-compat-data/pull/10629